core/config/project_settings.cpp

```diff
@@ -1342,6 +1342,9 @@ ProjectSettings::ProjectSettings() {
 	GLOBAL_DEF_RST(PropertyInfo(Variant::FLOAT, "audio/general/2d_panning_strength", PROPERTY_HINT_RANGE, "0,2,0.01"), 0.5f);
 	GLOBAL_DEF_RST(PropertyInfo(Variant::FLOAT, "audio/general/3d_panning_strength", PROPERTY_HINT_RANGE, "0,2,0.01"), 0.5f);
 
+	GLOBAL_DEF(PropertyInfo(Variant::INT, "audio/general/ios/session_category", PROPERTY_HINT_ENUM, "Ambient,MultiRoute,PlayAndRecord,Playback,Record,SoloAmbient"), 0);
```

Comment:
```suggestion
	GLOBAL_DEF(PropertyInfo(Variant::INT, "audio/general/ios/session_category", PROPERTY_HINT_ENUM, "Ambient,Multi Route,Play And Record,Playback,Record,Solo Ambient"), 0);
```

Reply:
Should like be "Play and Record" following the most common US title case rules (unless it's actually known as "Play And Record" in the iOS API).

Reply:
They use camel case for the audio session constants like playAndRecord. I think it's fine to adapt it to common case rules.